### DISCUSSION TITLE
Any recommendations for how to keep Karabiner.ts file sync'ed across multiple computers?

### DISCUSSION THREAD

**Discussion by sinewave** [evan-liu/karabiner.ts#133] [Q&A]
I use multiple MacBooks and would like to keep my config file sync'ed across all machines so I can easily make edits and tweaks 😄 

I'd prefer to use something like iCloud or Dropbox if possible as it would be harder for me to setup GitHub at work. 

Is it as easy as following the setup instructions, e.g. `npx create-karabiner-config@latest` inside of my iCloud/Dropbox folder or could that cause issues?

**Comment by evan-liu** [comment]
If your config is not too complex, and you can log in to GitHub at work, how about [fork the online editor](https://stackblitz.com/fork/github/evan-liu/karabiner.ts/tree/main/editor?file=rules.js)?

**Reply by sinewave** [reply]
Thx, I tried using the online editor but it seems to crash the editor (it's approx 5k+ lines) 😬

**Reply by evan-liu** [reply]
In that case, yes, just putting your project inside iCloud or Dropbox should be fine. If you'd like to only backup your config file you can write a script to sync the files to and from iCloud/Dropbox.
